cosmos/kima_testnet.json

```diff
@@ -24,7 +24,8 @@
       {
         "coinDenom": "KIMA",
         "coinMinimalDenom": "uKIMA",
-        "coinDecimals": 6
+        "coinDecimals": 6,
+        "coinGeckoId": "kima"
```

Comment:
This one as well!

Reply:
I thought I removed that? Must have pushed the wrong file - let me check.